Gruntfile.js

```diff
@@ -0,0 +1,69 @@
+module.exports = function(grunt) {
+    var BUNDLE_NAME = 'examplenews',
```

Comment:
If you use another name for your bundle than `examplenews` don't forget to change it the variables `BUNDLE_NAME` and `name` in this two files.